application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -42,8 +45,17 @@ public TagsCommand(TagSystem tagSystem) {
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
         // TODO A list might be better than comma separated, which is hard to read
-        event.replyEmbeds(MessageUtils.generateEmbed("All available tags",
-                String.join(", ", tagSystem.getAllIds()), event.getUser(), TagSystem.AMBIENT_COLOR))
+        ArrayList<String> list = new ArrayList<>(tagSystem.getAllIds());
+
+        event
+            .replyEmbeds(MessageUtils.generateEmbed("All available tags",
+                    IntStream.range(1, list.size())
+                        .filter(x -> x % 10 == 0 || x == list.size() - 1)
+                        .mapToObj(i -> String.join(", ",
+                                i % 10 != 0 && i != list.size() - 1 ? list.subList(i - 10, i)
+                                        : list.subList((i / 10) * 10, i + 1)))
+                        .collect(Collectors.joining("\n")),
```

Comment:
What is this doing? A comment should clarify and explain it to readers.

Also, there arte a lot of "magic values" (in particular a lot of `10`s), what do they mean? Use some properly named constants instead.

Also think about extracting this logic into a helper method, if applicable.